playbooks/edx_sandbox.yml

```diff
@@ -33,6 +33,8 @@
   roles:
     - role: swapfile
       SWAPFILE_SIZE: 4GB
+    - role: aws
```

Comment:
We generally use the `playbooks/hms_app.yml` file to deploy HMS, since we've customized that one for HMS.
Though this role is already included in that file, but it doesn't use the conditional `COMMON_ENABLE_AWS_INTEGRATION`.

Though perhaps we could re-integrate these two as the original reason for splitting into it's own file is no longer important. We'd only done this to split off rabbitmq into it's own instance.